activate.sh

```diff
@@ -17,8 +17,8 @@ autoenv_init() {
 	# We do this in a subshell so we can cd/chdir
 	_files="`
 		_hadone=''
-		while [ "${PWD}" != "${_mountpoint}" ]; do
-			_file="${PWD}/${AUTOENV_ENV_FILENAME}"
+		while [ "$(pwd -P)" != "${_mountpoint}" ]; do
+			_file="$(pwd -P)/${AUTOENV_ENV_FILENAME}"
```

Comment:
Love to hear that it works for you! If you could remove that one call for `df` and modify the changelog, I can merge it :+1: